common/transport/lwm2m/src/main/java/org/thingsboard/server/transport/lwm2m/server/uplink/DefaultLwM2mUplinkMsgHandler.java

```diff
@@ -150,7 +150,7 @@
     private final LwM2MTelemetryLogService logService;
     private final LwM2mTransportServerHelper helper;
     private final TbLwM2MDtlsSessionStore sessionStore;
-    private final LwM2mClientContext clientContext;
+    public final LwM2mClientContext clientContext;
```

Comment:
if you need to use clientContext in some tests you can add (inject) this class instead of changing the modificator.

Reply:
OK